integration_tests/src/main/python/hdfs_utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from pathlib import Path
+
+def hdfs_glob(path_str, pattern):
```

Comment:
I suggest putting these APIs to src/main/python/conftest.py near to spark_tmp_path, and adding comment for why they are created and when to use them.